src/widgets/BadgeSet.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, View, TouchableHighlight, Text } from 'react-native';
+import { Badge } from 'react-native-elements';
+import { SUSSOL_ORANGE } from '../globalStyles';
+import Popover from 'react-native-popover-view';
+
+export class BadgeSet extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVisible: false,
+      buttonRect: {},
+    };
+  }
+
+  showPopover() {
+    this.refs.button.measure((ox, oy, width, height, px, py) => {
```

Comment:
![j0e0p3iiic](https://user-images.githubusercontent.com/683452/52106983-bdfe2500-261c-11e9-9380-9c8d75c575e5.gif)

As you can see in the trace output, 
- ox and oy are relative x and y positions to the wrapper.
- px and py are absolute position relative to the entire view.

This is done because the popover actually overlays over the entire screen, it even as a transparent background that covers the entire screen which you click on to make the popover go away.

Reply:
Ah that's cool. Thanks!